### PR TITLE
163 cr adding a new field into events table 2

### PR DESCRIPTION
**Pull request by IsharaEkanayaka**

### CODE REVIEW

File: resources/views/backend/event/create.blade.php [hunk 1]
```diff
@@ -65,6 +65,76 @@
...
+                        }
+                    });
+                </script>
+
+
+
+
+
+
+
```
Comment (NuwanJ) [inline resources/views/backend/event/create.blade.php:137]: Can you try something like this? It is simple and robust

https://dev.to/snehalkadwe/how-to-use-select-2-with-multiple-options-in-laravel-and-livewire-4p4a (Hope you can do alphine)

Or you can reuse the item adder with some small changes.

**Review by NuwanJ — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/Backend/EventController.php [hunk 2]
```diff
@@ -21,7 +21,8 @@ class EventController extends Controller
     public function create()
     {
         try {
-            return view('backend.event.create');
+            $events = Event::all();
```
[the remark this answers is not in the document]
Comment (IsharaEkanayaka) [reply app/Http/Controllers/Backend/EventController.php:24]: it is not needed. removed it

File: app/Http/Controllers/Backend/EventController.php [hunk 3]
```diff
@@ -76,6 +82,10 @@ public function store(Request $request)
     public function edit(Event $event)
     {
         try {
+
+            $eventTypes = $event->event_type;
```
[the remark this answers is not in the document]
Comment (IsharaEkanayaka) [reply app/Http/Controllers/Backend/EventController.php:86]: thanks for pointing. It is not used anywhere

**Review by NuwanJ — APPROVED** [review]
Approved with minor UI changes

**Pull request merged**